src/test/opentracing_api.ts

```diff
@@ -0,0 +1,79 @@
+
+import { expect } from 'chai';
+import * as opentracing from '../index';
+
+export function opentracingAPITests(): void {
```

Comment:
Q: assuming we keep ot.Tracer as a concrete class, why can't these checks be in the api-compatibility file?